packages/Path-View-Core.package/SClassDiagram.class/instance/dragBottomBy..st

```diff
@@ -0,0 +1,5 @@
+geometry
+dragBottomBy: aValue
+
+	self extent: self extent + (0 @ aValue)
```

Comment:
if i´m not mistaken, we update the scrollpanes extent within the classDiagrams extent-method. so we have to keep it that way or overwrite 'withBottom' as well.